massa-ledger-exports/src/key.rs

```diff
@@ -1,58 +1,77 @@
 use massa_models::{
-    address::{Address, AddressDeserializer, ADDRESS_SIZE_BYTES},
+    address::{Address, AddressDeserializer},
     serialization::{VecU8Deserializer, VecU8Serializer},
 };
-use massa_serialization::{DeserializeError, Deserializer, SerializeError, Serializer};
+use massa_serialization::{Deserializer, SerializeError, Serializer};
 use nom::error::{ContextError, ParseError};
 use std::ops::Bound::Included;
 
 pub const BALANCE_IDENT: u8 = 0u8;
 pub const BYTECODE_IDENT: u8 = 1u8;
 pub const DATASTORE_IDENT: u8 = 2u8;
 
-/// Balance key formatting macro
-#[macro_export]
-macro_rules! balance_key {
-    ($addr:expr) => {
-        [&$addr.prefixed_bytes()[..], &[BALANCE_IDENT]].concat()
-    };
+#[repr(u8)]
+#[derive(PartialEq, Eq, Clone, Debug)]
+pub enum KeyType {
+    BALANCE = 0,
+    BYTECODE = 1,
+    DATASTORE(Option<Vec<u8>>) = 2,
```

Comment:
Would be nice to have some doc explaining what is Option<Vec<u8>> here (I guess this is the datastore key?). Any reason why we have an Option here?

Reply:
This wasn't used anymore. I removed it in [this commit](https://github.com/massalabs/massa/pull/3455/commits/a7da4a7c68bdbe92ce65bb5ff2bd842ba87122c7).